chapter09/axp/ex03

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env perl
+
+use 5.014;
+use warnings;
+use Regexp::Assemble;
+
+my $fname = $ARGV[0] // 'patterns.dat';
+my $finish = $^O eq "MSWin32" ? "<Ctrl>+Z" : "<Ctrl>+D";
+
+open my $file, $fname or die "Couldn't read $fname: $!\n";
+my $regex = Regexp::Assemble->new->add( <$file> )->re;
```

Comment:
This is just the good ol' [readline operator](http://perldoc.perl.org/perlop.html#I%2fO-Operators) passing an array to `add`.